lib/fork.js

```diff
@@ -16,9 +16,25 @@ module.exports = function (file, opts) {
 		} : false
 	}, opts);
 
+	var env = process.env;
+
+	if (process.env.NODE_PATH) {
+		var nodePath = process.env.NODE_PATH
+			.split(path.delimiter)
+			.map(function (p) {
+				return path.resolve(p);
+			})
+			.join(path.delimiter);
```

Comment:
Could compute this outside of the function so it can be reused for subsequent forks.